readme.md

```diff
@@ -82,7 +93,10 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+var eat_bamboo = Panda.prototype = {
+  var self = this;
+  self.num_bamboo_eaten += 1;
+}
```

Comment:
A few misconceptions here.  We want to add a new method to the prototype, not assign the prototype to the a variable.  The reason we use `var self = this;` is to ensure that we have access to the context outside of this function. We don't need that here.  Also, assigning it within the function is useless - we want self to be a different context.
```js
Panda.prototype.eat_bamboo = function() {
  this.num_bamboo_eaten = this.num_bamboo_eaten + 1;
}
```